tests/unit/support/AsyncStorage.ts

```diff
@@ -21,7 +19,15 @@ function delayOperation(operation: Function, operationName: string) {
 	};
 }
 
-const createAsyncStorage = createInMemoryStorage.mixin({
+const createAsyncStorage = compose(InMemoryStorage).mixin({
```

Comment:
This is a temporary hack as I haven't updated this test implementation(on this branch). It's just wrapping the class in a call to compose to make it a factory so it can be extended.